jsettlers.mapcreator/src/jsettlers/exceptionhandler/localization/labels_de.properties

```diff
@@ -0,0 +1,29 @@
+#Generated by ResourceBundle Editor (http://essiembre.github.io/eclipse-rbe/)
+###############################################################################
+# Copyright (c) 2015
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+# DEALINGS IN THE SOFTWARE.
+###############################################################################
+#
+# ** Tips to translate **
+# May use http://essiembre.github.io/eclipse-rbe/ as editor, makes it a lot easyer
+# Use a dot (.) to group tags by Category, this is common in property files, and then
+# it's displayed as groups in the Translator plugin mentioned above
+
+action.close = Dialog schliessen
+action.exit  = Applikation beenden
```

Comment:
I did this to a custom file because I thought it will may be moved. And in the swing branch its already moved, the files should be there merged together.